docs/aws/audit/shieldmonitoring/rules/aws_shield_drt_access.mdx

```diff
@@ -23,6 +23,254 @@ CBP
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can download it from the official AWS website and configure it using the "aws configure" command. You will need to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. Once the AWS CLI is set up, you can use the "aws shield describe-protection" command to list the details of the AWS Shield protections for your account. This command will return a JSON output with the details of the protections.
+
+   Command: 
+   ```
+   aws shield describe-protection --protection-id <your-protection-id>
+   ```
+   Replace `<your-protection-id>` with the ID of the protection you want to check.
+
+3. To check if the Shield Response Team (SRT) has access to your AWS account, you can use the "aws shield describe-assistance" command. This command will return a JSON output with the details of the AWS Shield assistance for your account.
+
+   Command: 
+   ```
+   aws shield describe-assistance --region <your-region>
+   ```
+   Replace `<your-region>` with the region where your AWS Shield is configured.
+
+4. If the SRT has access to your AWS account, the "aws shield describe-assistance" command will return a JSON output with the "emergencyAssistancePlan" field set to "ACTIVE". If the SRT does not have access to your AWS account, the "emergencyAssistancePlan" field will be set to "INACTIVE".
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3):
+   First, you need to install and configure AWS SDK for Python (Boto3). You can install it using pip:
+   ```
+   pip install boto3
+   ```
+   Then, configure your AWS credentials. You can configure your credentials either by setting the following environment variables:
+   ```
+   AWS_ACCESS_KEY_ID = 'your_access_key'
+   AWS_SECRET_ACCESS_KEY = 'your_secret_key'
+   ```
+   Or, you can create the credential file yourself. By default, its location is at `~/.aws/credentials`. At a minimum, the credentials file should look like this:
+   ```
+   [default]
+   aws_access_key_id = YOUR_ACCESS_KEY
+   aws_secret_access_key = YOUR_SECRET_KEY
+   ```
+
+2. Use Boto3 to check Shield Response Team (SRT) access:
+   You can use the `describe_subscription` method from the Shield client in Boto3 to check if SRT has access to your AWS account. Here is a sample script:
+   ```python
+   import boto3
+
+   # Create a Shield client
+   client = boto3.client('shield')
+
+   # Describe the subscription
+   response = client.describe_subscription()
+
+   # Check if SRT access is enabled
+   if 'AutoRenew' in response and response['AutoRenew'] == 'ENABLED':
+       print("SRT has access to the AWS account.")
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>